assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6

```diff
@@ -317,6 +317,11 @@ export default createWidget("discourse-reactions-actions", {
           return false;
         } else if (reaction.id === post.current_user_reaction.id) {
           post.reactions[index].count -= 1;
+          const users = post.reactions[index].users;
+          users.splice(
+            users.indexOf(users.findBy("username", "ahmedgagan6")),
```

Comment:
? 😅

Reply:
when a user drops a reaction, we were not removing the reaction-user, so removed it here

Reply:
read the code again please :p

Reply:
You hardcoded your username: ahmedgagan6

Reply:
oooh got it😂  a real mess😜